blocks/v2-images-grid/v2-images-grid.css

```diff
@@ -0,0 +1,95 @@
+:root {
+  --v2-images-grid-gutter: 8px;
+  --v2-images-grid-big-height: 168px;
+  --v2-images-grid-small-height: 125px;
+  --v2-images-grid-padding-space: 24px;
+}
+
+.redesign-v2 .v2-images-grid-container .v2-images-grid-wrapper {
+  padding: 0 100px;
+}
+
+.v2-images-grid-container .with-marker::before {
```

Comment:
IMO the heading shouldn't be part of the block. We should create a section with the heading and `v2-images-grid`. To add the marker to the heading we could use the section metadata class `header with mark` (it is implemented in the scope of the cards - we should merge the cards PR first :P ).

BTW the `with-marker::before` has a width already declared and we should not change it. On the design you can find many widths for the heading marker but we decided to simplify it to have one width everywhere (one for mobile and one for desktop).

Reply:
I didn't check that it's part of card's block. I check if we already have something in place, we didn't and saw testimonial block has similar approach. Will remove these styles